mu/virtual_environment.py

```diff
@@ -622,6 +692,7 @@ def create(self):
         self.install_baseline_packages()
         self.register_baseline_packages()
         self.install_jupyter_kernel()
+        self.settings["mu_version"] = mu_version

```

Comment:
Just to confirm, will this end up in `settings.json` or `venv.json`? If it's `settings.json` then I think this will be the only setting in that file (right now), everything else is in `session.json`.

Reply:
Playing devils' advocate for the "other half" of the "convinced" stakes.... having everything in a single json file means you only have one place to look for such configuration-y stuff.

Reply:
Ok; I'm going to stick with the separate `venv.json` for now simply because I don't want to change too much. But I'm definitely open to a later merge of the various files.